variants/t-echo/variant.h

```diff
@@ -206,6 +206,9 @@ External serial flash WP25R1635FZUIL0
 // To debug via the segger JLINK console rather than the CDC-ACM serial device
 // #define USE_SEGGER
 
+// T-Echo does not have a free serial port for this module
```

Comment:
We do have serial.override_console_serial_port that lets you use the NMEA and Cal Topo modes to stream sentences over USB. Maybe we can gate those off with a finer grained macro? 
Exclude serial is still a good one to have for really small platforms like STM32 possibly?

Reply:
I didn't consider that. Will revert.